stl/inc/system_error

```diff
@@ -438,9 +440,9 @@ _CRTIMP2_PURE unsigned long __CLRCALL_PURE_OR_CDECL _Winerror_message(
 // CLASS _Generic_error_category
 class _Generic_error_category : public error_category { // categorize a generic error
 public:
-    _Generic_error_category() noexcept {
-        _Addr = _Generic_addr;
-    }
+    constexpr _Generic_error_category() noexcept : error_category(_Generic_addr) {}
+
+    constexpr _Generic_error_category(const uintptr_t _Addr_) noexcept : error_category(_Addr_) {}

```

Comment:
I observe we could combine these constructors into single constructor with a default argument - but I think making the second `explicit` would be better.